examples/event-sources/azure-events-hub.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: EventSource
 metadata:
-  name: azure-events-hub-event-source
+  name: azure-events-hub
 spec:
   type: azureEventsHub
```

Comment:
There's not a `type` field in EventSource, and we don't need it.

Reply:
removed from examples